SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psd1

```diff
@@ -1,5 +1,5 @@
 @{
     ModuleVersion = '1.0'
     RootModule = 'SecretManagement.LastPass.Extension.psm1'
-    FunctionsToExport = @('Set-Secret','Get-Secret','Remove-Secret','Get-SecretInfo','Test-SecretVault')
+    FunctionsToExport = @('Set-Secret','Get-Secret','Remove-Secret','Get-SecretInfo','Test-SecretVault','Invoke-lpass')
```

Comment:
`Invoke-lpass` get exposed to the root module, but it doesn't bubble up to the end user.